lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -653,6 +661,14 @@ class ClassMetadataInfo implements ClassMetadata
      */
     public $changeTrackingPolicy = self::CHANGETRACKING_DEFERRED_IMPLICIT;
 
+    /**
+     * READ-ONLY: A Flag indicating whether or not one or more columns of this class have to be reloaded
+     * after insert / update operations.
+     *
+     * @var bool
+     */
+    public $requiresFetchAfterUpdate = false;
```

Comment:
You need to add support for this new property in serialization code here, should be something like:

```php
if ($this->requiresFetchAtfterUpdate) {
    $serialized[] = 'requiresFetchAtfterUpdate';
}
```